library/built-in/_prefabs/ControlNet/prefab_cnet_depth.ts

```diff
@@ -91,7 +91,14 @@ export const ui_subform_Depth_Zoe = () => {
 }
 
 // 🅿️ Depth RUN ===================================================
-export const run_cnet_Depth = async (Depth: OutputFor<typeof ui_subform_Depth>, cnet_args: Cnet_args, image: IMAGE) => {
```

Comment:
no need for async in those prefabs, since it's a synchronous operations that simply add nodes that will only later be executed